homeassistant/components/devolo_home_control/sensor.py

```diff
@@ -129,34 +107,37 @@ def __init__(self, homecontrol, device_instance, element_uid, consumption):
         """Initialize a devolo consumption sensor."""
         self._device_instance = device_instance
 
-        self.value = getattr(
+        self._device_class = DEVICE_CLASS_MAPPING.get(consumption)
+        self._sensor_type = consumption
+
+        super(  # pylint: disable=bad-super-call
+            DevoloMultiLevelDeviceEntity, self
+        ).__init__(
```

Comment:
```suggestion
        super().__init__(
```

Is there a reason you can't call `super()` directly?

Reply:
That would run DevoloMultiLevelDeviceEntity.\_\_init\_\_ and with that execute code I don't want to be executed. I want DevoloDeviceEntity.\_\_init\_\_ to run and I want to make use of the properties defined in DevoloMultiLevelDeviceEntity. So I thought the easiest way is to call the grandparent's \_\_init\_\_.

Reply:
Untested, but I think it will work to use `DevoloDeviceEntity.__init__` instead.

Reply:
Since my solution and your solution somehow looked like workarounds I hopefully solved it by making the deviation clearer.